src/nikonmn_int.cpp

```diff
@@ -1910,9 +1905,38 @@ namespace Exiv2 {
 //"data from TAG 0x98" "ltyp" " "TC" "MID" "maker" "PN" "lens name from manuf";
 //
 //------------------------------------------------------------------------------
-// Nikkor lenses by their LensID
+// EXIF data necessary for new entries
 //------------------------------------------------------------------------------
 //
+// lid:   LensIDNumber
+// stps:  LensFStops
+// focs:  MinFocalLength
+// focl:  MaxFocalLength
+// aps:   MaxApertureAtMinFocal
+// apl:   MaxApertureAtMaxFocal
+// lfw:   MCUVersion
+// ltype: LensType
+//
+// The tcinfo, dblid and mid fields are being ignored.
+//
+// Please note that all fields except LensType have to be looked up in the
+// Exif.NikonLd* prefix and not other Exif.Nikon* prefixes. For example: for modern
+// Nikon bodies with modern lenses, there will be both a Exif.Nikon3.LensFStops and a
+// Exif.NikonLd3.LensFStops entry in the EXIF data. You are looking for
+// Exif.NikonLd3.LensFStops.
+//
+// In most cases the necessary hex values should be extracted from a test image using
+// the following command:
+//
+// exiv2 -ph -g NikonLd3.LensIDNumber -g NikonLd3.LensFStops \
+// -g NikonLd3.MinFocalLength -g NikonLd3.MaxFocalLength \
+// -g NikonLd3.MaxApertureAtMinFocal -g NikonLd3.MaxApertureAtMaxFocal\
+// -g NikonLd3.MCUVersion -g Nikon3.LensType test.NEF
+//
+//------------------------------------------------------------------------------
+// Nikkor lenses by their LensID
+//------------------------------------------------------------------------------
```

Comment:
There already is a [wiki entry](https://github.com/Exiv2/exiv2/wiki/Adding-a-new-lens) that explains adding a lens. 

So maybe just point to that one from here?

Reply:
Right, I did not notice the wiki entry. Probably we can use this information to improve the wiki and then just add here a link to the wiki.

Reply:
I think all documentation that is necessary to continue developing the code should be part of the the release tarball. The wiki is not part of the release. So I think it should rather be the other way around: The Wiki should have a link to this file, or there should be a document in `docs/` that has the information.

Reply:
You've made a good point @Sturmflut . On the other hand, nowadays, anyone contributing to the project must have an internet connection and therefore access to the Wiki.

I have a slight preference for moving these notes to the wiki. The lenses topic is quite involved and not unique to the file `src/nikonmn_int.cpp`. What are your thoughts @hassec ? I can see you were the last one editing the wiki page.

Reply:
Right now the documentation situation is admittedly not that great as it is a bit spread across `md` files in the root and `doc/` dir as well as the wiki. 
I'd prefer to have it in the wiki too, as that's where we already have info about "adding a lens".  

Having the doc in the source file, IMHO, kind of defeats the purpose a bit because the user would have to already know a lot of implementation details to ever find this documentation

Reply:
I took the freedom to extend the wiki page with the information provided here (I hope this is fine for you @Sturmflut ):
https://github.com/Exiv2/exiv2/wiki/Adding-a-new-lens

I would suggest to keep lines 1911-1920 which provided relevant information in the context of understanding the fields of the structure. And replace replace the other lines for a reference to the wiki page. 

Would that work for you guys?